Assets/MixedRealityToolkit/Utilities/Gltf/Schema/GltfAccessorSparseIndices.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+
+namespace Microsoft.MixedReality.Utilities.Gltf.Schema
```

Comment:
Should this be namespace Microsoft.MixedReality **.Toolkit.** Utilities.Gltf.Schema?